certbot/tests/cli_test.py

```diff
@@ -352,12 +352,31 @@ def test_option_was_set(self):
         self.assertTrue(cli.option_was_set(key_size_option, key_size_value))
         self.assertTrue(cli.option_was_set('no_verify_ssl', True))
 
+        self.setUp() #cleanup
+        ecdsa_key_size_option = 'ecdsa_key_size'
+        ecdsa_key_size_value = cli.flag_default(ecdsa_key_size_option)
+        self.parse('--ecdsa-key-size {0}'.format(ecdsa_key_size_value).split())
+
+        self.assertTrue(cli.option_was_set(ecdsa_key_size_option, ecdsa_key_size_value))
+
         config_dir_option = 'config_dir'
         self.assertFalse(cli.option_was_set(
             config_dir_option, cli.flag_default(config_dir_option)))
         self.assertFalse(cli.option_was_set(
             'authenticator', cli.flag_default('authenticator')))
 
+        self.setUp() #cleanup
+        key_type_option = 'key_type'
+        key_type_value = cli.flag_default(key_type_option)
+        self.parse('--key-type {0}'.format(key_type_value).split())
+        print("key type: {}".format(key_type_option))
```

Comment:
The lint failure is because on python 2, print is a built-in keyword used like this `print 'Hello World!'`, while on python 3, this is a built-in function called like this `print('Hello World!').

And pylint is executed with python 2 in our CI, so it complains about the superfluous parenthesis on that version. 

Add `from __future__ import print_function` to instruct python 2 interpreter to also use print as a built-in function, and pylint will stop to complain. 

Or you can also just remove the two prints here. Most of the time printing from unit tests is not useful. Either the test passes and pytest will hide every output, or what is printed is significant for the test success, and an assertion should be use instead. Printing is then useful only for debugging purpose when a test fails.